src/vendor/overrides.scss

```diff
@@ -8,8 +8,44 @@ $font-family-sans-serif: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-s
 
 // Bootstrap
 .container-fluid {
-  padding-bottom: 30px;
-  max-width: 1376px;
-  padding-left: 0;
-  padding-right: 0;
+  // padding-bottom: 30px;
+  max-width: 1280px;
+  // padding-left: 20px;
+  // padding-right: 20px;
 }
+
+html {
+  height: 100%;
+}
+
+body {
+  height: 100%;
+}
+
+// todo: probably want to move this to the header component
+// but for now i'm just making changes here so the logos in
+// the header and the footer line up at every viewport width
```

Comment:
Yeah, we definitely want to move stuff off of `vendor`.  We can leave that for the next issue to tackle: https://github.com/openedx/frontend-app-library-authoring/issues/72